src/chatbots/actions.ts

```diff
@@ -203,43 +197,50 @@ const actions = [
     name: 'profile',
     execute: async (context: Context): Promise<string> => {
       const { args, reply } = context;
-
+      let user: User;
+      let username: string;
       // Check, if username was sent
       if (args.length !== 1) {
-        return reply(dictionary.BOT_MESSAGES.INCORRECT_INPUT, context);
-      }
-
-      // Get User from username
-      const username: string = args[0].toLowerCase();
-      const user: User = Cache.loadUser(username);
-
-      if (!user) {
-        return reply(
-          dictionary.BOT_MESSAGES.USERNAME_NOT_FOUND(username),
-          context,
-        );
-      }
-
-      const submissionsButtion: ReplyMarkupCommand = {
-        text: `${constants.EMOJI.SCROLL} Submissions`,
-        action: `/submissions ${username}`,
-      };
-
-      const avatarButton: ReplyMarkupCommand = {
-        text: `${constants.EMOJI.PEOPLE} Avatar`,
-        action: `/avatar ${username}`,
-      };
-
-      const ratingButton: ReplyMarkupCommand = {
-        text: `${constants.EMOJI.BACK_ARROW} Back to Rating`,
-        action: '/rating',
-      };
-
-      context.options.reply_markup = generateReplyMarkup({
-        buttons: [submissionsButtion, avatarButton, ratingButton],
-      });
-
-      return reply(dictionary.BOT_MESSAGES.USER_TEXT(user), context);
+        // Add user buttons
+        context.options.reply_markup = generateReplyMarkup({
+          buttons: createButtonsFromUsers({ action: 'profile' }),
+          isClosable: true,
+        });
+  
+        return reply(dictionary.BOT_MESSAGES.USER_LIST_USERS, context);
+      } else {
+          // Get User from username
+          username = args[0].toLowerCase();
+          user = Cache.loadUser(username);
+
+          if (!user) {
+            return reply(
+              dictionary.BOT_MESSAGES.USERNAME_NOT_FOUND(username),
+              context
+            )
```

Comment:
This repo is suited for working with ESLint. Please, set up ESLint to fix these small details

This should be:
```
return reply(
  dictionary.BOT_MESSAGES.USERNAME_NOT_FOUND(username),
  context,
)
```